raiwidgets/requirements.txt

```diff
@@ -1,10 +1,4 @@
-numpy>=1.17.2
```

Comment:
I see numpy being imported in cohort.py in raiwidgets. Why are we removing this direct dependency from requirements of this package?

Reply:
raiwidgets depends on responsibleai which depends on numpy already.  Specifying it multiple times is unnecessary duplication.

Reply:
Hmm if it was a slightly different setup (e.g., the two packages not in the same repo) I would object. In this particular case I guess it's fine since we control both dependency files.

Reply:
I still feel that direct imports should be hard dependencies (we could you unpin these for reducing complexity).

But major concern that I have here is from dev point of view there is very little sense I can make out of being able to import from numpy when it is not a direct dependency. I am not a big fan of implicit dependency as such.

Reply:
done, I changed the code back to keep the duplicate dependencies